isign/archive.py

```diff
@@ -150,7 +150,7 @@ def archive(cls, containing_dir, output_path):
             temp_zip_dir = tempfile.mkdtemp(prefix="isign-zip-")
             temp_zip_file = join(temp_zip_dir, 'temp.zip')
             call([get_helper('zip'), "-qr", temp_zip_file, "."])
-            shutil.move(temp_zip_file, output_path)
```

Comment:
I don't approve of this change - if we're going to make output paths relative to the cwd it should be in option parsing